CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+-   Added `enableCrossOrigin` prop to `Tumbnail` component
```

Comment:
Just to be a pain in the ass, this can be renamed to
'isCrossOriginEnabled'

Reply:
Yep, missed that one too 🙈